### PR TITLE
aws - security hub - enable bigger batching to improve write speeds

### PR DESCRIPTION
**Pull request by JohnHillegass**
For #6783, I have a test written for this with a 50 batch but recordings are huge. Does it look fine as is or want me to write one for a smaller batch like 3-5?

### CODE REVIEW

**Comment by JohnHillegass** [comment]
UI looks fine since we are keeping it to one resource per finding. This is only focused on increasing the IOPS.

image

File: c7n/resources/securityhub.py [hunk 1]
```diff
@@ -398,43 +398,45 @@ def process(self, resources, event=None):
...
-                        'value': '{}:{}'.format(
-                            finding['Id'], created_at)},
-                        self.manager).process(resource_set)
-                else:
-                    stats['Update'] += len(resource_set)
-
+        for resource_set in chunks(resources, batch_size):
+            findings = []
+            for k, v in self.group_resources(resource_set).items():
+                for resource in v:
```
Comment (kapilt) [inline c7n/resources/securityhub.py:404]: maybe keep v as grouped_resources

**Review by kapilt — COMMENTED** [review]
lgtm, one minor

**Comment by kapilt** [comment]
one thing to keep in mind, that there's a pending future enhancement needed for security hub,  we also need to change out the api call for import depending on the use case to support closing/updating extant findings.

File: c7n/resources/securityhub.py (hunk 1 above)
Comment (JohnHillegass) [reply c7n/resources/securityhub.py]: Sounds good, definitely is more explicit. Updated in c97ec42.

**Comment by JohnHillegass** [comment]
> one thing to keep in mind, that there's a pending future enhancement needed for security hub, we also need to change out the api call for import depending on the use case to support closing/updating extant findings.

Oh, interesting! Reading more about this now. I didn't realize the limitation was there on updating finding fields before.

**Comment by JohnHillegass** [comment]
Bumping https://github.com/cloud-custodian/cloud-custodian/issues/5606 as a reminder to look at reworking this for `BatchUpdateFindings`

**Review by kapilt — APPROVED** [review]
lgtm, thanks

**Pull request merged**